website/partners/templates/partners/partner.html

```diff
@@ -25,8 +25,14 @@ <h1 class="text-center section-title">{{ partner.name }}</h1>
                     <h4 class="mt-4 text-center text-md-left">{% trans "Address" %}</h4>
                     <p class="text-center text-md-left">
                         {{ partner.address }}<br/>
+                        {% if partner.adress2 %}
+                            {{ partner.address2 }}<br/>
+                        {% endif %}
                         {{ partner.zip_code }}<br/>
-                        {{ partner.city }}
+                        {{ partner.city }} <br />
+                        {% if partner.country != "NL" %}
+                            {{ partner.get_country_display }}
```

Comment:
For international students it may actually be useful to always show the country, since they do not know all the Dutch cities?